src/lib/components/CmsForm.svelte

```diff
@@ -186,81 +186,88 @@
-			</li>
-			<li>
-				To include pictures, add a paragraph with only <code class="font-mono"
-					>[Picture=slug]
-				</code>. You can also set the width, height and fit:
-				<code class="font-mono">[Picture=slug width=100 height=100 fit=cover]</code> or
-				<code class="font-mono">[Picture=slug width=100 height=100 fit=contain]</code>
-			</li>
-			<li>
-				To include challenges, add a paragraph with only <code class="font-mono"
-					>[Challenge=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your challenge
-			</li>
-			<li>
-				To include sliders, add a paragraph with only <code class="font-mono">[Slider=slug]</code>,
-				where
-				<code class="font-mono">slug</code> is the slug of your slider. You can specify the autoplay
-				duration in milliseconds like this:
-				<code class="font-mono">[Slider=slug?autoplay=3000]</code>
-			</li>
-			<li>
-				To include a specification widget, add a paragraph with only <code class="font-mono"
-					>[Specification=slug]</code
-				>, where <code class="font-mono">slug</code> is the slug of your specification.
-			</li>
-			<li>
-				To include a tag widget, add a paragraph with only <code class="font-mono">[Tag=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your tag. You can specify the display
-				option like this:
-				<code class="font-mono">[Tag=slug?display=var-1]</code>
-			</li>
-			<li>
-				To include a tagProducts widget, add a paragraph with only <code class="font-mono"
-					>[TagProducts=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your tag. You can specify the display
-				option like this:
-				<code class="font-mono">[TagProducts=slug?display=img-3]</code>
-			</li>
-			<li>
-				To include a form widget, add a paragraph with only <code class="font-mono"
-					>[Form=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your form.
-			</li>
-			<li>
-				To include a countdown widget, add a paragraph with only <code class="font-mono"
-					>[Countdown=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your countdown.
-			</li>
-			<li>
-				To include a specification widget, add a paragraph with only <code class="font-mono"
-					>[Specification=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your specification.
-			</li>
-			<li>
-				To include a gallery widget, add a paragraph with only <code class="font-mono"
-					>[Gallery=slug]</code
-				>, where
-				<code class="font-mono">slug</code> is the slug of your gallery. You can specify the display
-				option like this:
-				<code class="font-mono">[Gallery=slug?display=var-1]</code>
-			</li>
-		</ul>
+		<label class="checkbox-label my-2 block">
+			<input type="checkbox" name="showTips" class="form-checkbox peer" />
+			Show tips
+			<ul class="text-gray-700 my-3 list-disc ml-4 hidden peer-checked:block">
```

Comment:
Don't put everything in the `<label>`.

If `peer-has-checked` does not work (maybe we need to upgrade tailwind) go back to the previous version